setup.cfg

```diff
@@ -49,12 +48,12 @@ install_requires =
     distributed >= 2021.10.0
     holidays >= 0.13
     numpy >= 1.21.0
-    pandas >= 1.3.0
+    pandas >= 1.4.0, != 1.4.2
```

Comment:
Do we need to bump the minimum pandas and woodwork versions for the base install?